terraform/fullnode/variables.tf

```diff
@@ -74,6 +74,12 @@ variable "fullnode_helm_values" {
   default     = {}
 }
 
+variable "fullnode_helm_values_list" {
+  description = "List of values to pass tp public fullnode, for setting different value per node"
```

Comment:
nit typo: `tp` --> `to` :p

Reply:
Also update description so that user knows `length(fullnode_helm_values_list)` must equal `var.num_fullnodes` ?